projects/fundamental-ngx/src/lib/calendar/calendar.component.ts

```diff
@@ -0,0 +1,485 @@
+import { Component, EventEmitter, Input, OnInit, Output, HostListener, ElementRef, SimpleChanges } from '@angular/core';
+
+export type DatePickerType = 'single' | 'range';
```

Comment:
Since this type applies to both calendar and date picker, maybe it should be called "CalendarType"?

Reply:
Agreed